include/belle.hh

```diff
@@ -1196,10 +1196,15 @@ private:
 
   private:
 
+#ifndef _MSC_VER
     // generic lambda for sending different types of responses
-    static auto const constexpr send = [](auto self, auto&& res) -> void
+    static auto constexpr send = [](auto self, auto&& res) -> void
+#else
+    template <typename SELF_T, typename RES_T>
+    static constexpr void send(SELF_T self, RES_T&& res)
+#endif
     {
-      using item_type = typename std::remove_reference<decltype(res)>::type;
+      using item_type = std::remove_reference_t<decltype(res)>;
```

Comment:
This is nice, less boilerplate. Since it's unrelated to the Visual Studio fix, would it be better separated into its own commit or pull request?

Reply:
Sorry, I'm not so firm with the github pull request mechanism. Realized to late that I need a new branch for every request.